examples/blue-green-deployment/README.md

```diff
@@ -0,0 +1,98 @@
+# Blue/Green deployment example for PostgreSQL
```

Comment:
Please add link to this example in the main README.md file in the root.

Reply:
Updated in https://github.com/terraform-aws-modules/terraform-aws-rds/pull/517/commits/b959874cdf337e3220347a81862e17b34cf64946.